SofaKernel/modules/SofaRigid/JointSpringForceField.h

```diff
@@ -37,6 +37,16 @@ namespace component
 namespace interactionforcefield
 {
 
+//This class is specialized with cudatypes
+template<class DataTypes>
```

Comment:
For curiousity I searched for the specialization but didn't find it. Is it in a private plugin ?